hack/cluster-deploy.sh

```diff
@@ -68,7 +68,8 @@ fi
 # Deploy infra for testing first
 _kubectl create -f ${MANIFESTS_OUT_DIR}/testing
 
-# Do not deploy cdi-operator for sriov-lane until kubevirt/kubevirt#3850 is fixed
+# Do not deploy any cdi-operator related objects on sriov-lane
+# until kubevirt/kubevirt#3850 is fixed
```

Comment:
- Is #3850 relevant now? I think it needs to be dropped.

- Can we place all the `if` body in  a function? It will be much more readable.

Reply:
There were changes to #3850 it used to describe the issues with cdi on sriov-lane, I will mention this issue https://github.com/kubevirt/kubevirt/issues/4120 its the right one.